api/v1alpha1/base_types.go

```diff
@@ -266,6 +266,8 @@ type S3 struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	TLS *TLS `json:"tls,omitempty"`
+
+	Prefix string `json:"prefix"`
```

Comment:
Could you documment and annotate this field like the other please?

Reply:
Sure thing!